src/Calendar.php

```diff
@@ -110,6 +110,15 @@ public function addEvent(EventInterface $event)
         return $this;
     }
 
+    public function addEvents(array $events)
```

Comment:
Forgot that, sense i almost never add them in my own projects sorry for that will take care of that. when i impliment your suggestions.